app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
Rewrite this part, according to Checklist

Reply:
Not agree, according to README.md SoftwareEngineer class should have one method: learn_skill that takes a skill of string type and should add it to the skills list. And checklist example of bad practice is about duplicating code and adding miltiple values, not about using append or expand methods of list. Below condition is from checklist you mentioned: 
```Use the extend() method or += operator to add multiple values to a single list```

Reply:
My bad

Reply:
no problem